rln/benches/poseidon_tree_benchmark.rs

```diff
@@ -0,0 +1,71 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use rln::{
+    circuit::{Fr, TEST_TREE_HEIGHT},
+    hashers::PoseidonHash,
+};
+use utils::{FullMerkleTree, OptimalMerkleTree, ZerokitMerkleTree};
+
+pub fn get_leaves(n: u32) -> Vec<Fr> {
+    (0..n).map(|s| Fr::from(s)).collect()
+}
+
+pub fn optimal_merkle_tree_poseidon_benchmark(c: &mut Criterion) {
+    c.bench_function("OptimalMerkleTree::full_height_gen", |b| {
```

Comment:
```suggestion
    c.bench_function("OptimalMerkleTree::<Poseidon>::full_height_gen", |b| {
```
to differentiate between the benches in the utils crate (they use keccak, this one uses poseidon.)